tests/containers/kubectl.pm

```diff
@@ -29,7 +29,7 @@ sub run {
     record_info("kubectl", $version);
     if ($version !~ /v\Q$k8s_version\E/) {
         # NOTE: Remove when bsc is resolved
-        if ($k8s_version eq "1.23" || $k8s_version eq "1.26") {
+        if (is_sle("<15-SP6") && $k8s_version =~ /^1\.2[36]$/) {
```

Comment:
the k8s regexp is too tricky to identify the real numbers.
I'd suggest to simply integrate the existing expression, like:
```suggestion
        if (is_sle("<15-SP6") && ($k8s_version eq "1.23" || $k8s_version eq "1.26")) {
```

Reply:
ah: automerge running .... 🏃🏼